ts/screens/profile/EmailInsertScreen.tsx

```diff
@@ -0,0 +1,284 @@
+/**
+ * A screen where user after login (with CIE) can set email address if it is
+ * not present in the profile.
+ */
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { EmailString } from "@pagopa/ts-commons/lib/strings";
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { Content, Form, Text as NBText, View } from "native-base";
```

Comment:
"View" should come from react-native, not from native-base

Reply:
fixed in [c3a5abe](https://github.com/pagopa/io-app/pull/4303/commits/c3a5abefcb7ff68a1366f6cdf23e8a588076691f)